app/forms/waste_carriers_engine/contact_name_form.rb

```diff
@@ -4,10 +4,12 @@ module WasteCarriersEngine
   class ContactNameForm < BaseForm
     attr_accessor :first_name, :last_name
 
+    validates :first_name, :last_name, "waste_carriers_engine/person_name": true
+
     def initialize(transient_registration)
       super
-      self.first_name = @transient_registration.first_name
-      self.last_name = @transient_registration.last_name
+      self.first_name = transient_registration.first_name
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.first_name = transient_registration.first_name
```